app/main.py

```diff
@@ -1,3 +1,52 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.customer import Customer
+from app.shop import Shop
+import json
```

Comment:
Order and group your imports to follow PEP-8 import rules here and in the other files